process/smartContract/process.go

```diff
@@ -1581,10 +1581,11 @@ func (sc *scProcessor) createSCRForSenderAndRelayer(
 	callType vmcommon.CallType,
 ) (*smartContractResult.SmartContractResult, *smartContractResult.SmartContractResult) {
 	if vmOutput.GasRefund == nil {
+		// TODO: compute gas refund with reduced gasPrice if we need to activate this
 		vmOutput.GasRefund = big.NewInt(0)
 	}
 
-	gasRefund := core.SafeMul(vmOutput.GasRemaining, tx.GetGasPrice())
+	gasRefund := sc.economicsFee.ComputeFeeForProcessing(tx, vmOutput.GasRemaining)
```

Comment:
what happens if a transaction will start to be processed just before the enable epoch and the gasRefund process is done just after the enable epoch?

Reply:
some tokens will be burnt